books/free-programming-books-fr.md

```diff
@@ -240,6 +241,11 @@
 * [Apprendre Meteor](https://mquandalle.gitbooks.io/apprendre-meteor/content/) - Maxime Quandalle
 
 
+### Pascal - Object Pascal
+
+* [J'apprends à programmer en Pascal Objet](https://lazarus.developpez.com/livre/livre-pascal-objet/) - Gilles Vasseur, Jean-Luc Gofflot
```

Comment:
not free.

Reply:
if the book is creative commons licensed by the author, it can be uploaded to Internet Archive, which will distribute it without requiring an email address or account.

Reply:
Sorry for this one, I haven't seen the form before the download, i'm a user of this forum and am connected. Line changed.